src/main/kotlin/com/github/pool_party/pull_party_bot/Utils.kt

```diff
@@ -0,0 +1,64 @@
+package com.github.pool_party.pull_party_bot
+
+/**
+ * Two char sequence Levenshtein distance computation.
+ */
+fun CharSequence.levenshtein(
+    rhs: CharSequence,
+    deleteCost: Int = 1,
+    insertCost: Int = 1,
+    replaceCost: Int = 1,
+    transposeCost: Int = 1
```

Comment:
We also need to figure out the best parameters

Reply:
Or just dga